Tests/sniff-examples/empty_non_variable.php

```diff
@@ -42,5 +42,20 @@
 empty();
 empty( /*comment*/ );
 
+
+// Issue #210 + some extra variants - should all be ok as well.
+empty(${$a . $b });
```

Comment:
No specific code style here, more than anything having varying spacing in the test cases is a way to test that the sniff works independently of code style.